pom.xml

```diff
@@ -12,16 +11,69 @@
     <description>A microservice for storing associations between user IDs and DIDs for applications that want to use DID-Auth</description>
 
     <properties>
-        <swagger2markup.version>1.3.1</swagger2markup.version>
+        <java.version>11</java.version>
         <asciidoc.input.directory>${project.basedir}/src/docs/asciidoc</asciidoc.input.directory>
         <swagger.output.dir>${project.basedir}/src/main/resources/swagger</swagger.output.dir>
         <asciidoc.snippet.directory>${project.build.directory}/asciidoc/snippets</asciidoc.snippet.directory>
         <asciidoc.generated.directory>${project.build.directory}/asciidoc/generated</asciidoc.generated.directory>
         <swagger.input>${swagger.output.dir}/swagger.sdk.json</swagger.input>
+
+
+        <!-- Docker settings -->
+        <docker.registry.push>hostname</docker.registry.push> <!-- Create <server> section in .m2/chainSettings.xml with this registry
```

Comment:
Please include this type of info in the readme as well. You are talking there about several commands to create docker images and push them through a registry, so please explain there with references to the pom

Reply:
Fixed in upcomming commit